_i18n/en/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,131 @@
+2) The `Restore wallet` page will be displayed:
 
-On the next page, give your wallet a strong password and confirm it before clicking the `Right` arrow:
+![restore-from-seed](/img/resources/user-guides/en/restore_account/restore-from-seed.png)
+
+3) On `Wallet name`, enter the name of the wallet you want to create. You don't need to use the same name of the wallet being restored.
+
+4) On `Wallet location`, choose where you want the wallet file to be saved.
+
+5) Keep `Restore from seed` selected.
+
+6) On the next field, type your 25 word `mnemonic seed`. It is expected that some mnemonic seeds have repeated words, as they work as a verification for errors (checksum).
+
+7) Use the field `Seed offset passphrase` to enter your passphrase, if you have one. When you enter a passphrase, Monero GUI will use your mnemonic seed together with your passphrase to compute a new wallet.
+
+8) The next field allows you to enter a `Restore height`. Even though this field is optional, it's not recommended to leave it blank, since setting a restore height makes your wallet synchronization faster. The wallet restore height is a setting that tells Monero GUI at which block height it should start scanning the blockchain looking for transactions. Ideally it should be the number of the block which included the first incoming transaction to your wallet:
+
+![restore-height-field](/img/resources/user-guides/en/restore_account/restore-height-field.png)
+
+If you don't know the block number of the first incoming transaction in your wallet, you can also enter the date or an estimated date (YYYY-MM-DD format):
+
+![restore-height-field-date](/img/resources/user-guides/en/restore_account/restore-height-field-date.png)
+
+If you set a restore height higher than the block height of your first incoming transaction, Monero GUI will not detect all transactions of your wallet. In consequence, your balance will be wrong and Monero GUI will not work properly.
+
+If you set a restore height lower than the block height of your first incoming transaction, Monero GUI will scan unnecessary blocks. In consequence, your initial wallet synchronization will take longer than necessary, but Monero GUI will still work.
+
+Therefore, if you're unsure about the block height of your first incoming transaction, it's always better to set a lower restore height instead of a higher one.
+
+![restore-height](/img/resources/user-guides/en/restore_account/restore-height.png)
+
+After entering your restore height, click on `Next` button.
+
+9) On the next page, give your wallet a strong password and confirm it. Since your wallet will hold your private keys and seed, it is really important to enter a strong password. The more you mix up letters (upper-case and lower-case), numbers, and symbols, the more secure your password is, and the harder it is for a brute force attack to discover it. It is NOT recommended to use a blank password or weak passwords like "password123" or "abcd1234".
```

Comment:
```suggestion
9) On the next page, give your wallet a strong password and confirm it. Since your wallet will hold your private keys and seed, it is really important to enter a strong password.
```

It's not in the scope of this guide to give suggestion about password creation. Better leave that to the app themselves.